apache/noservice.shtml

```diff
@@ -9,20 +9,20 @@
                         Your Preview is Almost Ready
                     </h1>
                 </div>
-                <p class="intro">
-                    You are almost there! We just need a bit more information before Tugboat can build previews for your
-                    repository. Run through the following checklist and you will be on your way!
-                </p>
+                <p class="intro"></p>
```

Comment:
Did you mean to remove this tag as well?

Reply:
I did mean to remove that line. It's not really needed since there is no checklist. Thoughts?

Reply:
I like having an intro text. It would be nice to actually tell what's going on. In this case services are for sure the issue as they're not found. Maybe something like this?

> You are almost there! Before Tugboat can build previews, one or more services need to be enabled and configured.

And then right below is the how-to on that.